.circleci/config.yml

```diff
@@ -140,12 +140,6 @@ workflows:
   untagged-build:
     jobs:
       - install
-      - lint:
```

Comment:
this does not change anything, and we already doing the build and the lint in parallel so we are good

Reply:
Yeah, I realized that after I saw it build. We'll just use this PR to test and we can close it after